plugins/chat/db/migrate/20240422042830_add_excerpt_to_chat_messages.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+class AddExcerptToChatMessages < ActiveRecord::Migration[7.0]
+  def change
+    add_column :chat_messages, :excerpt, :string, limit: 1000, null: true
```

Comment:
I intentionally left a higher limit here than we have set within in the Message model to allow for some flexibility in future. Open to ideas here though

Reply:
What will happen for all the existing messages with no existing excerpt?

Reply:
They would not have an excerpt, only new messages or those being updated will generate the excerpt. I am wondering if we want to bother generating these as it could potentially be tens of thousands in some sites.

If we want to do prevent showing messages without an excerpt, then we could fall back to the raw_excerpt method on the model as a temporary measure. Then remove it later after a couple of weeks.

Reply:
> If we want to prevent showing messages without an excerpt, then we could fall back to using the raw_excerpt within the serializer when excerpt is blank. Then remove it later after a couple of weeks.

Yes I would suggest we do this

Reply:
I've added this to the message serializer which prevents this issue on older messages